### ISSUE TITLE
Display ROM information in ROM selector

### ISSUE THREAD

**Issue by eku** [DS-Homebrew/TWiLightMenu#889]
**Is your feature request related to a problem? Please describe.**

Theme: 3DS
Currently the ROM selector only shows information for NDS ROM, for all others only the file name, which is usually too long to fit into the popup window.

**Describe the solution you'd like**

For all ROM of Nintendo games it should be possible to extract and display the game information, just like for NDS.

**Describe alternatives you've considered**

none

**Additional context**

The one written here is for the 3DS theme, but it could be the same for the other themes.

**Comment by NightYoshi370** [comment]
Do the other ROMs even have titles and stuff like that?

**Comment by eku** [comment]
No titles, only file names.

**Comment by NightYoshi370** [comment]
If they contain titles

**Comment by eku** [comment]
I found  only [rominfo](https://github.com/moriczgergo/rominfo) that is OS independent. Here's the output
```
$ rominfo -f Mario\ \&\ Luigi\ -\ Superstar\ Saga.gba 
Console: GameBoy Advance
Title: MARIO&LUIGIP
Maker code: 01
Game info: Older game
Region: Europe
```
If you know tools for NES and SNES roms I would test my roms with these.

**Comment by Darknior** [comment]
For sure there are these info in ROM but it is already very long to scan a game directory, we already lost a ton of time t stand for directory refresh very time we change ... because TWilight read roms to show icon ... if you add this option and it make more longer to load a directory, please add it as an optional option :p Thanks

**Comment by eku** [comment]
@Darknior I agree with you in principle. However, this is only a problem if all ROMs are in one folder, which is not the case for me and probably others.

**Comment by Darknior** [comment]
Yes for sure but many do like me too. @eku more directories more hard to found a game ... maybe not for you but children that d'ont read fine and not read english too ... having only one dir with all the same console games is the best way to search a game. But i understand your point of view.
But now i have try to browse without "image cash," it's faster and the lag is when the DS read the image. The DS can read to at same time the game informations ;)

**Comment by khatharr** [comment]
I'd prefer if you could add custom icons and titles in the same way that you can currently add boxarts.

**Comment by eku** [comment]
This leads me to the question why all boxarts are in a folder in the subfolder of TWL instead of in subfolders of the folder of the ROMs, like the game saves.

**Comment by RocketRobz** [comment]
> This leads me to the question why all boxarts are in a folder in the subfolder of TWL instead of in subfolders of the folder of the ROMs, like the game saves.

The box art are named as the ROM's TIDs, so that it'll be shown for those ROMs containing that TID, as well as Slot-1 cartridges.
